<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="2" failed="0" total="2" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="31" started-at="2018-05-19T01:52:51Z" finished-at="2018-05-19T01:52:51Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="31" started-at="2018-05-19T01:52:51Z" finished-at="2018-05-19T01:52:51Z">
      <class name="application.testcases.LoginTestScenarios4">
        <test-method status="PASS" signature="setupReport()[pri:0, instance:application.testcases.LoginTestScenarios4@96532d6]" name="setupReport" is-config="true" duration-ms="425" started-at="2018-05-18T21:52:50Z" finished-at="2018-05-18T21:52:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupReport -->
        <test-method status="FAIL" signature="startBrowser(java.lang.String)[pri:0, instance:application.testcases.LoginTestScenarios4@96532d6]" name="startBrowser" is-config="true" duration-ms="0" started-at="2018-05-18T21:52:51Z" finished-at="2018-05-18T21:52:51Z">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Parameter 'Browser' is required by @Configuration on method startBrowser but has not been marked @Optional or defined
in F:\automation_code\selenium_pageobject\com.learnselenium.framework\XMLFiles\CRM.xml]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Parameter 'Browser' is required by @Configuration on method startBrowser but has not been marked @Optional or defined
in F:\automation_code\selenium_pageobject\com.learnselenium.framework\XMLFiles\CRM.xml
	at org.testng.internal.Parameters.createParameters(Parameters.java:154)
	at org.testng.internal.Parameters.createParameters(Parameters.java:361)
	at org.testng.internal.Parameters.createConfigurationParameters(Parameters.java:84)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:201)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:178)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startBrowser -->
        <test-method status="SKIP" signature="loginWithValidCredentials()[pri:0, instance:application.testcases.LoginTestScenarios4@96532d6]" name="loginWithValidCredentials" duration-ms="0" started-at="2018-05-18T21:52:51Z" description="This test will verify user is able to login with valid crendentials" finished-at="2018-05-18T21:52:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginWithValidCredentials -->
        <test-method status="SKIP" signature="tearDown(org.testng.ITestResult)[pri:0, instance:application.testcases.LoginTestScenarios4@96532d6]" name="tearDown" is-config="true" duration-ms="0" started-at="2018-05-18T21:52:51Z" finished-at="2018-05-18T21:52:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="SKIP" signature="logOut()[pri:0, instance:application.testcases.LoginTestScenarios4@96532d6]" name="logOut" duration-ms="0" started-at="2018-05-18T21:52:51Z" description="Logout user" depends-on-methods="application.testcases.LoginTestScenarios4.loginWithValidCredentials" finished-at="2018-05-18T21:52:51Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method LoginTestScenarios4.logOut()[pri:0, instance:application.testcases.LoginTestScenarios4@96532d6] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method LoginTestScenarios4.logOut()[pri:0, instance:application.testcases.LoginTestScenarios4@96532d6] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logOut -->
        <test-method status="SKIP" signature="generateReport()[pri:0, instance:application.testcases.LoginTestScenarios4@96532d6]" name="generateReport" is-config="true" duration-ms="0" started-at="2018-05-18T21:52:51Z" finished-at="2018-05-18T21:52:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- generateReport -->
      </class> <!-- application.testcases.LoginTestScenarios4 -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
